tests/calculations/functions/test_create_kpoints_from_distance.py

```diff
@@ -0,0 +1,56 @@
+# -*- coding: utf-8 -*-
+"""Tests for the `create_kpoints_from_distance` calculation function."""
+from aiida.orm import Bool, Float
+
+from aiida_quantumespresso.calculations.functions.create_kpoints_from_distance import create_kpoints_from_distance
+
+
+def test_kpoints_00(generate_structure):
+    """Test `create_kpoints_from_distance` calculation function.
+
+    Case: cubic cell with all sides equal and all periodic.
+    Expected result: isotropic kpoint mesh.
+    """
+    structure = generate_structure('silicon-kinds')
+
+    distance = Float(0.3)
+    force_parity = Bool(False)
+
+    kpoints = create_kpoints_from_distance(structure, distance, force_parity).get_kpoints_mesh()[0]
+
+    assert kpoints[0] == kpoints[1] == kpoints[2]
+
+
+def test_kpoints_01(generate_structure):
```

Comment:
I think this and the test below are not really testing anything different when it comes to the `create_kpoints_from_distance` function? `test_kpoints_02` is more explicit, so I'd keep that one and remove this test.